### ISSUE TITLE
Make a Team for Face Recognition Engine with Accord.Net and let's build our OWN SDK/API

### ISSUE THREAD

**Issue by blaisexen** [accord-net/framework#1779]
well be using 
MulticlassSupportVector with IRadialBasisKernel(RBF)
MultinomialLogisticRegression with BroydenFletcherGoldfarbShanno/ConjugateGradient
RestrictedBoltzmannMachine with ResilientBackpropagationLearning
DeepNeuralNetwork with ResilientBackpropagationLearning
ResilientBackpropagation(RproP)  

here's my 2 videos for image recognition,

store item recognition
http://ge.tt/7v3lwBp2

object recognition
http://ge.tt/38qK1eo2

Who's with me?

First member will be assigned to aligned the eyes of the face, BUT with the help of our discussion from our chat board, comments here? I think that's Ok.

**Comment by blaisexen** [comment]
anyway, if there is NON joining, then watchout my YOUTUBE videos of Face Recognition with OpenCv
https://www.youtube.com/channel/UCaH7B7KmJAe3ppesuLbcOGQ

But please join, at-least.

**Comment by lindomar-marques55** [comment]
deixe-me ver se entendi voce quer criar uma nov api de reconhecimento facial sem o accord.net? é isso mesmo? se for teriamos que entender de redes neurais e outras coisas mais.

**Comment by blaisexen** [comment]
Sim, eu entendo, e já fiz isso usando o Accord.net, mas algumas correções talvez o tornem bom como outros sdk de reconhecimento de rosto, como alinhamento de olhos e obtenção do tamanho de rostos. obrigado por perguntar.

**Comment by lindomar-marques55** [comment]
Hi. i'm setting up an ironman jarvis brand virtual assistant. I implemented voice recognition using microsoft speech and implemented facial recognition using emgucv with Haarcascade recognition but the results with emgucv didn't make me happy so I would like to use neural network because I read that it generates more accurate results but I still can't find an example Good to be able to study, you can give me one. The face detection part is already ok, what I need is an efficient and accurate mechanism to say that the face belongs to john and not to joseph. forgive my english but i am using google translator.

**Comment by blaisexen** [comment]
Hi,
this is Accord.net implementation using SVM,HMM,HCRF and NN
https://www.youtube.com/watch?v=tEUxnQ7EHZA

EmguCv is good only for Indoor Face Recognition,
https://www.youtube.com/watch?v=014hHoz7jXw

For Accord Face Recognition, here I have,
https://sourceforge.net/projects/face-detection-accord-net-csvb/

What I don't have is the face alignment between eyes, and may also include other face properties of face like nose, mouth distance.

I much time doing anything, but I could not give time coding the face alignment, if you have it please mail me, and after the right and good result I'll give you the full source code, but take note only 2 of us is with this.

thanks for your reply

----------------------------------------------------------------------------

Oi,
esta é a implementação do Accord.net usando SVM, HMM, HCRF e NN
https://www.youtube.com/watch?v=tEUxnQ7EHZA

O EmguCv é bom apenas para o reconhecimento de faces internas,
https://www.youtube.com/watch?v=014hHoz7jXw

Para o reconhecimento de rosto Accord, aqui estou eu,
https://sourceforge.net/projects/face-detection-accord-net-csvb/

O que não tenho é o alinhamento do rosto entre os olhos e também pode incluir outras propriedades do rosto, como nariz, distância da boca.

Eu demoro muito tempo fazendo alguma coisa, mas não pude dar tempo de codificar o alinhamento do rosto, se houver, envie-me um email e, após o resultado certo e bom, darei o código fonte completo, mas observe que apenas 2 de nós é com isso.

Obrigado pela sua resposta

**Comment by lindomar-marques55** [comment]
Using your example link 
https://sourceforge.net/projects/face-detection-accord-net-csvb/ 
caused me the blue screen of death when switching cameras. 
probably still have to stop the videodevice using signaltostop (). 

here is what (https://github.com/lindomar-marques55/lfm-cam-face ) I have implemented so far

**Comment by blaisexen** [comment]
Oi,
O que não tenho é o alinhamento do rosto entre os olhos e também pode incluir outras propriedades do rosto, como nariz, distância da boca.

Não tenho muito tempo fazendo nada, mas não pude dar tempo de codificar o alinhamento do rosto, se houver, envie-me um email e, após o resultado certo e bom, darei o código-fonte completo, mas tome nota apenas 2 de nós está com isso.

Obrigado pela sua resposta
